src/main/webapp/i18n/de/exerciseChatbot.json

```
@@ -7,7 +7,8 @@
             "accept": "Akzeptieren",
             "decline": "Ablehnen",
             "unreadMessages": "Ungelesene Nachrichten",
-            "buttonTooltip": "Beginne mit Iris zu chatten!"
+            "buttonTooltip": "Beginne mit Iris zu chatten!",
+            "firstMessage": "Hallo, ich bin Iris! Ich kann Ihnen bei Ihrer Programmieraufgabe helfen. Du kannst hier mehr über mich erfahren."
```

This would be a more consistent German translation, we want Iris to always use 'du' with the students:
```suggestion
            "firstMessage": "Hallo, ich bin Iris! Ich kann dir bei deiner Programmieraufgabe helfen. Du kannst hier mehr über mich erfahren."
```